examples/02_additive_2d_microstructure.py

```diff
@@ -20,12 +20,13 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 """
-Microstructure analysis
-#######################
+2D Microstructure analysis
+##########################
 
 This example shows how to use PyAdditive to determine
-the microstructure for a sample coupon with given material
-and machine parameters.
+the two-dimensional microstructure in the XY, XZ and YZ
```

Comment:
```suggestion
the two-dimensional microstructure in the XY, XZ, and YZ
```
We use Oxford commas, which merely means that you always place a comma before an "and" or an "or" in a list of more than two items.

Reply:
Agree about the Oxford comma. 
Also, I think we should stick to 2D and 3D instead of spelling out two-dimensional and three-dimensional. I can't find 2D and 3D specifically mentioned in the Google developer documentation style guide but it does say to use acronyms instead of spelling things out when they are universally understood. Furthermore, won't our example title pages say 2D and 3D?